templates/python/package/payload.py.twig

```diff
@@ -0,0 +1,96 @@
+import os
+import mimetypes
+import json
+import re
+
+class Payload:
+    def size(self):
+        if self.data:
+            return len(self.data)
+        if self.path:
+            return os.stat(self.path).st_size
+        raise Exception('Empty payload')
+
+    @classmethod
+    def handle_form_data(cls, boundary, response_body):
+        parts = response_body.split(boundary)
+        data = {}
+        for part in parts:
+            lines = [line for line in part.split('\r\n') if line]
+            if not lines:
+                continue
+            match = re.search(r'name="?(?P<name>\w+)', part)
+            if match:
+                name = match.group('name')
+                lines = lines[1:]  # Remove the first line (Content-Disposition)
+                if lines and lines[0] == 'Content-Type: application/json':
+                    lines = lines[1:]  # Remove the Content-Type line
+                    headers = json.loads(''.join(lines))
+                    data[name] = {header['name']: header['value'] for header in headers}
+                else:
+                    data[name] = ''.join(lines) if lines else ''
+
+        data['responseStatusCode'] = int(data.get('responseStatusCode', 0))
+        data['duration'] = float(data.get('duration', 0))
+        data['responseBody'] = cls.from_string(data.get('responseBody', ''))
+        return data
```

Comment:
@loks0n are people able to then do `response.responseBody.toJSON()` or `response.responseBody.toBinary()` for example ?

Reply:
`to_json()` and `to_binary()` but yeah